frontend/app/become-artisan/page.tsx

```
@@ -0,0 +1,141 @@
+            <Card className="max-w-3xl mx-auto border-none shadow-lg">
+                <CardHeader className="text-center space-y-2">
+                    <CardTitle className="text-3xl font-bold bg-gradient-to-r from-purple-600 to-blue-500 bg-clip-text text-transparent">Become an Artisan</CardTitle>
+                    <CardDescription className="text-lg">
+                        Join our vibrant community of creators and bring your craft to the world
+                    </CardDescription>
+                </CardHeader>
+                <CardContent className="px-8">
+                    <div className="space-y-8">
+                        <div className="bg-gradient-to-r from-purple-50 to-blue-50 p-6 rounded-xl">
+                            <h3 className="text-xl font-semibold flex items-center gap-2 text-purple-700">
+                                <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" strokeWidth="2" strokeLinecap="round" strokeLinejoin="round" className="lucide lucide-gift"><path d="M20 12v10H4V12" /><path d="M2 7h20v5H2z" /><path d="M12 22V7" /><path d="M12 7H7.5a2.5 2.5 0 0 1 0-5C11 2 12 7 12 7z" /><path d="M12 7h4.5a2.5 2.5 0 0 0 0-5C13 2 12 7 12 7z" /></svg>
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)